src/pages/index.js

```diff
@@ -74,170 +53,183 @@ const IndexPage = ({ data, pageContext }) => {
 
   return (
     <PageContext.Provider value={pageContext}>
-      <>
-        <SEO />
-        <PageTitle>Observability for every developer</PageTitle>
+      <SEO />
+      <PageLayout type={PageLayout.TYPE.SINGLE_COLUMN}>
+        <PageLayout.Header title="Observability for every developer" />
```

Comment:
This adds an underline to the title on the homepage, which doesn't currently exist. We might want to add a lil `css` override for this.

Reply:
I'm actually not seeing the underline for most of our pages. We may just want to remove this an add it where applicable.

Reply:
I added the underline intentionally on every page, including the homepage, so that every page had a consistent page title treatment. I might need to chat with @djsauble one more time and get his thoughts 😄